src/v0/destinations/facebook_conversions/config.js

```diff
@@ -0,0 +1,110 @@
+const { getMappingConfig } = require('../../util');
+
+const CONFIG_CATEGORIES = {
+  USERDATA: {
+    standard: false,
+    type: 'identify',
+    name: 'FBCUserDataConfig',
+  },
+  COMMON: { name: 'FBCCommonConfig' },
+  APPDATA: { name: 'FBCAppEventsConfig' },
+  SIMPLE_TRACK: {
+    standard: false,
+    type: 'simple track',
+    name: 'FBCSimpleCustomConfig',
+  },
+  PRODUCT_LIST_VIEWED: {
+    standard: true,
+    type: 'product list viewed',
+    eventName: 'ViewContent',
+    name: 'FBCProductListViewedCustomData',
+  },
+  PRODUCT_VIEWED: {
+    standard: true,
+    type: 'product viewed',
+    eventName: 'ViewContent',
+    name: 'FBCProductViewedCustomData',
+  },
+  PRODUCT_ADDED: {
+    standard: true,
+    type: 'product added',
+    eventName: 'AddToCart',
+    name: 'FBCProductAddedCustomData',
+  },
+  ORDER_COMPLETED: {
+    standard: true,
+    type: 'order completed',
+    eventName: 'Purchase',
+    name: 'FBCOrderCompletedCustomData',
+  },
+  PRODUCTS_SEARCHED: {
+    standard: true,
+    type: 'products searched',
+    eventName: 'Search',
+    name: 'FBCProductSearchedCustomData',
+  },
+  CHECKOUT_STARTED: {
+    standard: true,
+    type: 'checkout started',
+    eventName: 'InitiateCheckout',
+    name: 'FBCCheckoutStartedCustomData',
+  },
+  OTHER_STANDARD: {
+    standard: true,
+    type: 'otherStandard',
+    name: 'FBCSimpleCustomConfig',
+  },
+  PAGE_VIEW: {
+    standard: true,
+    type: 'page_view',
+    eventName: 'PageView',
+    name: 'FBCSimpleCustomConfig',
+  },
+};
+
+const MAPPING_CONFIG = getMappingConfig(CONFIG_CATEGORIES, __dirname);
+const ACTION_SOURCES_VALUES = [
+  'email',
+  'website',
+  'app',
+  'phone_call',
+  'chat',
+  'physical_store',
+  'system_generated',
+  'other',
+];
+
+const OTHER_STANDARD_EVENTS = [
+  'AddToWishlist',
+  'AddPaymentInfo',
+  'Lead',
+  'CompleteRegistration',
+  'Contact',
+  'CustomizeProduct',
+  'Donate',
+  'FindLocation',
+  'Schedule',
+  'StartTrial',
+  'SubmitApplication',
+  'Subscribe',
+];
+
```

Comment:
Why we are considering AddToWishlist and AddPaymentInfo as other standard events?
These events are supported as standard ecomm events as per our general ecomm doc
https://www.rudderstack.com/docs/event-spec/ecommerce-events-spec/#wishlist
https://www.rudderstack.com/docs/event-spec/ecommerce-events-spec/#ordering

Reply:
updated